src/Core/Resolvers/MsSqlQueryExecutor.cs

```diff
@@ -77,7 +77,7 @@ public MsSqlQueryExecutor(
                   httpContextAccessor,
                   handler)
         {
-            handler?.Subscribe(MSSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, OnConfigChanged);
+            handler?.Subscribe(MSSQL_QUERY_EXECUTOR_ON_CONFIG_CHANGED, MsSqlQueryExecutorOnConfigChanged);
```

Comment:
nit: perhaps this doesn't have latest changes from Aaron's order of operations PR. The function `MsSqlQueryExecutorOnConfigChanged` doesn't need the prefix `MsSqlQueryExecutor`

Reply:
Aaron's changes will be on top of my changes from this PR.

Reply:
We need to use this function only, it's created in the same class for proper reinitialization.

Same has been done in other executer class as well

Reply:
gotcha. We don't need to prefixes due to other changes I had Aaron make. I'll raise a separate PR.